### DISCUSSION TITLE
EOL for typegoose 10 and release for typegoose 11

### DISCUSSION THREAD

**Discussion by vasyl97kharchenko** [typegoose/typegoose#801] [Q&A] [answered]
Could you please tell me when is EOL(approximately) for typegoose 10 and the release date (approximately) for typegoose 11? I'm starting a new project and planning to use typegoose. Is it reasonable to use typegoose 10 for it or it's better to choose typegoose 11 (wait till it's released)?

**Comment by hasezoey** [comment, marked as answer]
> and the release date (approximately) for typegoose 11

i am targeting 27.03.23 (d/m/y), currently version 11 is in beta (current latest is `11.0.0-beta.2`) and i dont expect much to change anymore since the latest beta

> Could you please tell me when is EOL(approximately) for typegoose 10

depends on what the definition for EOL is, i wont be putting new features (or changes) to older versions unless specifically asked, aside from security and bug fixes

**Reply by vasyl97kharchenko** [reply]
Thanks a lot for your reply!
